content/it/community/_index.html

```diff
@@ -1,69 +1,237 @@
+<p>La comunità di Kubernetes - cioè gli utenti, i contributori e la cultura che abbiamo costruito tutti assieme - è una dei maggiori motivi della crescita esponenziale di questo progetto open-source. La nostra cultura e i nostri valori continuano a crescere e modificarsi con il progetto stesso. Lavoriamo tutti assieme per il miglioramento costante del progetto e nei modi di interazione.
+<br><br>Noi siamo le persone che aprono gli issue, le pull request, che partecipano alle riunioni delle SIG, i meetup di Kubernetes, e le KubeCon, che spingiamo l'innovazione e l'adozione di Kubernetes, che eseguiamo <code>kubectl get pods</code>, e che contribuiscono in moltissimi altri modi egualmente vitali al progetto. Vai avanti a leggere per scoprire come essere coinvolto in queste attività e diventare parte di questa meravigliosa comunità.</p>
+<br class="mobile">
+</div>
+
+<div class="navbar">
+
+<a href="#conduct">Codice di Condotta</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<a href="#videos">Video</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<a href="#discuss">Discussioni</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<a href="#events">Eventi e meetups</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<a href="#news">Notizie</a>
+
+</div>
+<br class="mobile"><br class="mobile">
+<div class="imagecols">
+<br class="mobile">
+  <div class="imagecol">
+    <img src="/images/community/kubernetes-community-final-03.jpg" alt="Immagine di una conferenza Kubernetes" style="width:100%" class="desktop">
+  </div>
+
+  <div class="imagecol">
+    <img src="/images/community/kubernetes-community-final-04.jpg" alt="Immagine di una conferenza Kubernetes" style="width:100%" class="desktop">
+  </div>
+
+  <div class="imagecol" style="margin-right:0% important">
+    <img src="/images/community/kubernetes-community-final-05.jpg" alt="Immagine di una conferenza Kubernetes" style="width:100%;margin-right:0% important" class="desktop">
+  </div>
+  <img src="/images/community/kubernetes-community-04-mobile.jpg" alt="Immagine di una conferenza Kubernetes" style="width:100%;margin-bottom:3%" class="mobile">
+
+<a name="conduct"></a>
+</div>
+
+
+
+<div class="conduct">
+<div class="conducttext">
+<br class="mobile"><br class="mobile">
+<br class="tablet"><br class="tablet">
+<div class="conducttextnobutton" style="margin-bottom:2%"><h1>Codice di Condotta</h1>
+La comunità di Kubernetes valorizza il rispetto e l'inclusività, e forza il Codice di Condotta in ogni interazione. Se noti una violazione del Codice di Condotta durante une evento o un meeting, in Slack, o in qualsiasi altra interazione, contatta la Kubernetes Code of Conduct Committee mandando una mail a <a href="mailto:conduct@kubernetes.io" style="color:#0662EE;font-weight:300">conduct@kubernetes.io</a>. Tutte le segnalazioni sono tenute confidenziali. Puoi leggere di più sulla Kubernetes Code of Conduct Committee <a href="https://github.com/kubernetes/community/tree/master/committee-code-of-conduct" style="color:#0662EE;font-weight:300">qui</a>.
```

Comment:
e forza il Codice di Condotta in ogni interazione -> e per mette il codice di condotta alla base di ogni iterazione
qualsiasi altra interazione -> qualsiasi altra communicazione